python/cudf_polars/cudf_polars/dsl/expr.py

```diff
@@ -1038,6 +1084,24 @@ def do_evaluate(
                 )
                 arg = evaluated.obj_scalar if evaluated.is_scalar else evaluated.obj
             return Column(plc.replace.replace_nulls(column.obj, arg))
+        elif self.name == "pow":
+            column = self.children[0].evaluate(df, context=context, mapping=mapping)
+            if isinstance(self.children[1], Literal):
+                arg = plc.interop.from_arrow(self.children[1].value)
```

Comment:
Why special-case the `Literal` case, given that `Literal.evaluate` just does what you have written here for you?